litmus-portal/frontend/src/components/Button/ButtonFilled/styles.ts

```diff
@@ -22,6 +22,12 @@ const useStyles = makeStyles((theme) => ({
       backgroundColor: theme.palette.primary.dark,
     },
   },
+  buttonTransparent: {
```

Comment:
why were these changes required...can you post screenshots of the button you are trying to make

Reply:
sure.

Reply:
cc: @S-ayanide

Reply:
as per my understanding of the designs...filled button doesn't need to be transparent.